scripts/build.sh

```diff
@@ -330,12 +330,14 @@ case "$product-$platform-$method" in
     "${firestore_emulator}" start
     trap '"${firestore_emulator}" stop' ERR EXIT
 
+    nproc=$(sysctl -n hw.ncpu)
     RunXcodebuild \
         -workspace 'Firestore/Example/Firestore.xcworkspace' \
         -scheme "Firestore_IntegrationTests_$platform" \
         -enableCodeCoverage YES \
+        -parallelizeTargets \
```

Comment:
Were you able to measure a speedup from this change?

Reply:
Actually this does not seem to help. It seems our xcode config does this by default. I reverted this change.